### DISCUSSION TITLE
[SOLVED] RSS Guard, the page in Wikipedia

### DISCUSSION THREAD

**Discussion by sakkamade** [martinrotter/rssguard#432] [General]
Deviated from https://github.com/martinrotter/rssguard/issues/430#issuecomment-853714787.

The progress (most likely only on completion) will be posted here.
If I'd need, I'll ask for some additional information, and then we shall eventually behold why, in all probabilities, it gets rejected.
And if it gets, the actual discussion shall commence there.

~~Most of information~~ All data for Wikipedia will naturally be extracted from [documentation](https://github.com/martinrotter/rssguard/blob/master/resources/docs/Documentation.md).

Estimated time for the first attempt to be sent as ready-to-read article: <2 weeks.

**Comment by martinrotter** [comment]
Well, I can say thank you, man. My attempts to write article were just blocked as I am the author. :( Even my account was blocked entirely as I was quite angry. 🗡️ 
If you have any questions, feel free to ask here. No need to rush anywhere.

**Reply by Korb** [reply]
Your emotions from your first experience of participating in improving Wikipedia completely coincide with mine, when in the early 2010s I tried to supplement articles about films with objective technical data, as provided by the template of articles of this type. I, too, was banned for spam. Since then, my opinion of Wikipedia hasn't been very high, let's say. I realized what kind of people write it and protect it from others.

**Comment by sakkamade** [comment]
Some draft may now be found here https://en.wikipedia.org/wiki/Draft:RSS_Guard. 
Although I can't seem to understand how to add references links, which appearing to be vital for the  review.

**Reply by martinrotter** [reply]
@sakkamade Nice, thanks, man.

I have almost zero experience with wikipedia, but [this](https://en.wikipedia.org/wiki/Help:Referencing_for_beginners) might help?

**Reply by martinrotter** [reply]
Couple of extra points:
- in some sentences, English is bit weird, but I am not native English speaker, so it could be fine:
    - "RSS Guard **is** released under GPL v3 license a"
    - "and using Qt what (**which**?) allows it to fit with t"
    - "synchronize online feeds like (**with**?) Tiny Tiny RSS, Nextcloud N"
    - "User can scrap**e** the data from w"
    - "RSS Guard **is** translated into many languages"

**Reply by martinrotter** [reply]
Adding references to RSS Guard documentation or other reference would certainly help. Other than that, it might be just okay. :)

**Comment by sakkamade** [comment]
> Couple of extra points: ...

Thank you, I'll take a look.

**Comment by sakkamade** [comment]
What was the date of initial release?

**Reply by sakkamade** [reply]
Well whatever, someone can add it afterwards.

**Comment by sakkamade** [comment]
The draft was submitted for review.

**Reply by martinrotter** [reply]
Sorry I was incredibly busy:

```
tag 1.9.9.3
Tagger: Martin Rotter
Date:   Tue Feb 4 19:13:39 2014 +0100
```

**Reply by sakkamade** [reply]
> ```
> tag 1.9.9.3
> Tagger: Martin Rotter
> Date:   Tue Feb 4 19:13:39 2014 +0100
> ```

Is this is the date of the very initial release? Or initial release on GitHub only?

**Comment by sakkamade** [comment]
So here we go: it was denied :heart: 

1. They say that there are not enough **reliable sources**
2. They say that there are not enough **reliable sources** which are **independent** of the subject

Hence if anyone will be so good as to find some articles about RSS Guard on some proper websites, I should expect that it will solve both.

**Reply by martinrotter** [reply]
@sakkamade There are virtually no reviews. It's kind of circled problem. One has to love Wikipedia! Some links below, you can certainly try, but it seems that RSS Guard would have to be much much more popular to justify its own wiki page.

https://www.softpedia.com/get/Internet/News-Newsgroups-Blog-Tools/RSS-Guard.shtml
https://robbinespu.gitlab.io/blog/2020/12/04/use-this-rss-guard/
https://www.linuxuprising.com/2021/02/rss-guard-is-qt-desktop-rss-feed-reader.html
https://ubunlog.com/en/rss-guard-feeds-ubuntu/

**Reply by sakkamade** [reply]
3. There is more:
    > Declining submission: v - Submission is improperly sourced and nn - Submission is about a topic not yet shown to meet general notability guidelines **(be more specific if possible) ([AFCH](https://en.wikipedia.org/wiki/Wikipedia:WikiProject_Articles_for_creation/Helper_script) 0.9.1)**

Huh?
Am I being asked to write the guidelines on how to install and use it??? (Go over that link...)

Oh well.

**Reply by martinrotter** [reply]
@sakkamade If I understand it correctly, AFCH is special script which you can install on your PC and which can check quality of your drafts. I don't think you should add install/use instructions about RSS Guard to the draft.

Perhaps you could use those "review" links and link to them in proper places where you write about RSS Guard features etc.

At this point it looks that adding the article is quite hard stuff, especially if the subject is not that "notable". Sadly, there is not enough information about RSS Guard written by "independent" authors. RSS Guard is not actually that well-known even in RSS community.

**Reply by sakkamade** [reply]
> special script which you can install on your PC and which can check quality of your drafts

Aw, indeed, I properly glanced over it just now. Whew.

**Reply by sakkamade** [reply]
> Perhaps you could use those "review" links

Yes, I have already used 3 of them, though I don't think I will use this one 

> https://robbinespu.gitlab.io/blog/2020/12/04/use-this-rss-guard/

**Reply by sakkamade** [reply]
> Sadly, there is not enough information about RSS Guard written by "independent" authors

Actually, I am surprised how you found these four. When I searched, I found none which may even somewhat be considered "reliable."

**Reply by sakkamade** [reply]
> > special script which you can install on your PC and which can check quality of your drafts
> 
> Aw, indeed, I properly glanced over it just now. Whew.

Still, what did they mean by 
> **(be more specific if possible)**

?

**Reply by sakkamade** [reply]
Well, I submitted it one more time... We'll see.

**Comment by sakkamade** [comment]
Well, I expected as much: Declined again:—

> This submission's references ... do not show significant coverage (not just passing mentions) about the subject in published, reliable, secondary sources that are independent of the subject. ...

---
Thus, I can do nothing until some well-known websites would publish articles of/including RSS Guard.

**Comment by sakkamade** [comment]
> **[UNSOLVABLE]** RSS Guard, the page in Wikipedia

Hahahaha, don't be so pessimistic ❤️

**Comment by sakkamade** [comment]
But indeed, it's unbelievable! 
Such lovely application with so earnest maintainer... And no duly articles about it can be found out there...

**Comment by sakkamade** [comment]
Ha! We are saved: https://itsfoss.com/feed-reader-apps-linux/
I will attempt submitting the page again, after some editing to-morrow.

EDIT: How could I miss it, I wonder.

**Reply by martinrotter** [reply]
I love how they screwed application name

>5. RSSGuard

No space.

**Reply by sakkamade** [reply]
Worry not!

> Note: The list is in no particular order of ranking.

**Comment by sakkamade** [comment]
Well, declined again.

**Comment by sakkamade** [comment]
However, this time I have actually received commentary from the reviewer!

> It reads like a website's "about us" page than presenting it in encyclopaedic manner. Please provide reliable sources independent of the subject to prove notability. Blogs and GitHub are sufficient to establish notability. GitHub may be used a link to repository, but cannot be accepted as a reliable source

So, all I can do at present is to remove linking to Github.

I still fail to understand what do they want that article to be. Gosh.

**Reply by sakkamade** [reply]
> I still fail to understand what do they want that article to be. Gosh.

I am of course talking about this: 

> Please provide **reliable sources independent of the subject to prove notability**

**Comment by sakkamade** [comment]
I have tried to address everything from that commentary except for reliable sources, since I can find none, was hoping that perhaps it will go well; however it appears that reliable sources are still lacking, and they are still a big issue.

---

> This submission is not adequately supported by reliable sources.

I expected for _itsfoss_ to be reliable enough, though. Albeit it might be considered a "passing mention", or wiki article truly needs to consist of a bunch of source links on Adobe inc. web-site, at this day and age :smile:

**Reply by martinrotter** [reply]
I have read some article on Wikipedia and basically every fact stated on the subject was linked to some external website supporting the claim.

So, basically everything state about features of RSS Guard must be backed up by link to some "reliable source", prefarably some 3rd-party review of RSS Guard, etc.

Some basic facts, like current version number, etc., can be safely taken from github, maybe from documentation.

**Reply by sakkamade** [reply]
Take a look at [wiki-page of GIMP](https://en.wikipedia.org/wiki/GIMP): Yes, it has quite a few source links, however there are large parts left "un-sourced", such as ["Selections and paths"](https://en.wikipedia.org/wiki/GIMP#Selections_and_paths), "Image editing", right below it, and a paragraph in "Development."

I see your point, but even so, I am much dubious that by removing parts without a source, the article will be accepted.

**Reply by martinrotter** [reply]
Yes, but it also states that the

<html><body>
<b>This article has multiple issues.</b> Please help <b><a class="external text" href="https://en.wikipedia.org/w/index.php?title=GIMP&amp;action=edit">improve it</a></b> or discuss these issues on the <b><a href="https://en.wikipedia.org/wiki/Talk:GIMP" title="Talk:GIMP">talk page</a></b>. <small><i>(<a href="https://en.wikipedia.org/wiki/Help:Maintenance_template_removal" title="Help:Maintenance template removal">Learn how and when to remove these template messages</a>)</i></small>
<div class="mw-collapsible-content" style="margin-top: 0.3em;">
      

This article needs additional citations for verification.  (December 2020)
--


</div>
</body>
</html>

Thing is that they do not accept new articles with some (minor) problems.

**Reply by sakkamade** [reply]
> Thing is that they do not accept new articles with some (minor) problems.

Hmm... I guess.
 I'll it try then.

**Reply by martinrotter** [reply]
Maybe use this as a base: https://en.wikipedia.org/wiki/Liferea

Structure the page literally the same way and use reliable linked source for each and every fact stated about the application.

For statistical facts (versions, date of release, initial release, maybe screenshots, link to documentation) - use github.

For facts about features use other sites - basically all sources independent from the subject (not written by me) should be relatively reliable - https://www.google.com/search?q=%22RSS+Guard%22&client=firefox-b-d&source=lnt&tbs=qdr:y&sa=X&ved=2ahUKEwjA4oH44KXyAhXC6aQKHTmcAB8QpwV6BAgBECU&biw=1488&bih=927

https://www.softpedia.com/get/Internet/News-Newsgroups-Blog-Tools/RSS-Guard.shtml
https://www.linuxuprising.com/2021/02/rss-guard-is-qt-desktop-rss-feed-reader.html
https://ubunlog.com/en/rss-guard-feeds-ubuntu/
https://www.linuxlinks.com/rss-guard-simple-light-easy-to-use-rss-atom-feed-aggregator/
https://techiestechguide.com/rss-guard-3-9-0-a-desktop-rss-feed-reader-for-ubuntu/
https://medevel.com/rss-readers-for-macos/

@sakkamade

**Reply by sakkamade** [reply]
> For facts about features use other sites - https://www.google.com/search...

The thing is that I could not bring myself to plant there everything I find through search engine. I tried to find **truly** reliable sources, but this is exactly what I ended up thinking: "Jesus, just link that thing to whatever you find."

**Reply by martinrotter** [reply]
:)

**Reply by martinrotter** [reply]
updated my previous comment, yes, this is jungle, it is not actually clear what they want, maybe it is that there is not enough information about rssguard from independent reliable source

**Reply by sakkamade** [reply]
> independent reliable source

To be precise, Independent _and_ reliable, or independent, reliable sources? 

But being that they say "is not adequately", I am inclined to former.

**Reply by martinrotter** [reply]
well, who decides if the website which posts some facts is "reliable", i believe that if the site looks "okay" and is independent from the subject, it is "reliable"

Look at liferea article

https://en.wikipedia.org/wiki/Liferea#cite_note-9

the author even uses "documentation" and blogs probably written by Liferea team too "LZone". Also, other referenced sites are LinuxInsider, Ubuntu Geek, Liferea.blogspot.com,  Launchpad.net - these are typical website to refer about linux projects.

**Reply by martinrotter** [reply]
what about using exactly same scheme as liferea page

## Introduction
- some basic facts about where development happens, what was initial version, when it was released, how often it is released - link to relevant parts of github project - all these statements are clearly foundable on github site

## Distinguishing features
- write about six, seven most killer features of the app - cite each feature to some independent website, preferably https://www.softpedia.com/get/Internet/News-Newsgroups-Blog-Tools/RSS-Guard.shtml and https://www.linuxuprising.com/2021/02/rss-guard-is-qt-desktop-rss-feed-reader.html

**Reply by sakkamade** [reply]
> well, who decides if the website which posts some facts is "reliable", i believe that if the site looks "okay" and is independent from the subject, it is "reliable"

Hmm, maybe you are right.
I am  visiting not so many web-sites, and very sparsely using the search engines to search for any web-site as source of true  information, for I understand what sort of sites are dominant there.

**Reply by martinrotter** [reply]
These are "fast-food" sites. They usually post rather short articles with not so much value. RSS Guard is not that known/popular to have some really really in-depth reviews out there, even negative ones.

**Reply by sakkamade** [reply]
> what was initial version, when it was released, how often it is released - link to relevant parts of github project - all these statements are clearly foundable on github site

If I understand them correctly, this is not whatever important therein.
And all this information is provided to wikipedia by another source, the page in which I have already created long ago: https://www.wikidata.org/wiki/Q107242660

**Reply by sakkamade** [reply]
> it is not actually clear what they want, maybe it is that there is not enough information about rssguard from independent reliable source

I am in much doubts that they are going through the web to search for sources of subject of article themselves, when verifying the article.

**Reply by sakkamade** [reply]
> > independent reliable source
> 
> To be precise, Independent _and_ reliable, or independent, reliable sources?
> 
> But being that they say "is not adequately", I am inclined to former.

Although before, there was:

> Please provide reliable sources independent of the subject

**Reply by sakkamade** [reply]
> Distinguishing features
write about six, seven most killer features of the app - cite each feature to some independent website, preferably 

This is not about quantity but quality, I have seen, though old, but some one-paragraph articles resided there just fine.

**Reply by sakkamade** [reply]
> what about using exactly same scheme as liferea page

It is not like my article is many different, but I believe I remove last three paragraphs in "Features", for I am unable to find anything to cite them to. 

EDIT: Hahaha. 
Funny how I search articles to refer to what I have already written.

**Reply by martinrotter** [reply]
Yes, just have everything factual cited from other (reliable) reviews, articles etc.

**Comment by sakkamade** [comment]
Honestly, never had I expected to make it this far: https://en.wikipedia.org/wiki/RSS_Guard

**Reply by martinrotter** [reply]
Holly sh**. OK, should I send any donations anywhere?

Also, I am blocked on wiki but made an appeal, so I will in the future enhance the page and also re-add RSS Guard to https://en.wikipedia.org/wiki/Comparison_of_feed_aggregators

Thank you once more.

**Reply by sakkamade** [reply]
> I will in the future enhance the page and also re-add RSS Guard to https://en.wikipedia.org/wiki/Comparison_of_feed_aggregators

Like so?

![image](https://user-images.githubusercontent.com/62730422/130138852-e83f3bb1-f9d3-4192-bb5c-52b6ac8fc944.png)

Sadly some kind little fairy already did exactly that. :sparkles:

**Reply by martinrotter** [reply]
Yes, thanks, there are some minor mistakes:

* RSS Guard does "support" Android (but user must compile for Android by himself and the overall UX will probably be shitty, maybe except for tablets with larger screens). Moreover, RSS Guard supports OS/2 - "Other" column).
* Delta updates - Not sure what "delta updates" is but I think it is actually new algorithm in Greader Plugin which is able to perform "delta" updates - in other words it only downloads trully not-yet-downloaded data. So maybe there should be "Yes (Google Reader API plugin only)".
* Also "HTTP Basic auth" is supported for respective plugins which use it (Nextcloud News, TT-RSS).

Other than that, good job! I will probably register on Wiki with different mail/username as current one is blocked and they already refused several appeals (I lost my temper).

**Reply by sakkamade** [reply]
For now, I decided simply to revert the part of revision where RSS Guard was removed, and see if it will be removed again (and its article to the boot).

> there are some minor mistakes

Thank you. I will address them.

> Moreover, RSS Guard supports OS/2 - "Other" column

Hm? What is it? OS/2 is already there.

**Comment by hercyle** [comment]
i saw the image in wikipedia, but how do i add these things in the _message viewer_ by this order?
- Written by Web Webster
- Title **/** _**URL**_ (this is neat. the title looks very crisp and i like that URL option separated by a <kbd>/</kbd>
- Gray line (separator)
- Date
- Description

![example](https://upload.wikimedia.org/wikipedia/commons/2/2d/RSS_Guard_4.2.1_%28dark_mode%29.png)

is it for windows users only and not linux users? in either case, i want that message viewer looks, Badly.
